src/content/contributing/community-membership/_index.en.md

```diff
@@ -0,0 +1,193 @@
+- [Open an issue][open issue] against the submariner-io/submariner repo
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of
+     your work on the project
+- Have your sponsoring committers reply confirmation of sponsorship: `+1`
+- Once your sponsors have responded, your request will be reviewed. Any missing
+  information will be requested.
+
+### Responsibilities and privileges
+
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly
+  transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+- They can be assigned to issues and PRs, and people can ask members for
+  reviews
+
+**Note:** Members who frequently contribute code are expected to proactively
+perform code reviews and work towards becoming a committer.
+
+## Committers
+
+Committers are able to review code for quality and correctness on some part of
+the project. They are knowledgeable about both the codebase and software
+engineering principles.
+
+**Until automation supports approvers vs reviewers:** They also review for
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** Entry in an CODEOWNERS file in a repo owned by the Submariner
+project.
+
+**Committer status is scoped to a part of the codebase.**
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a committer
+in an CODEOWNERS file:
+
+- Member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by two committers or project owners
+  - With no objections from other committers or project owners
+- May either self-nominate or be nominated by a committer/owner
+
+### Responsibilities and privileges
+
+The following apply to the part of codebase for which one would be a committer
+in an CODEOWNERS file:
+
+- Responsible for project quality control via [code reviews]
+  - Focus on code quality and correctness, including testing and factoring
+  - **Until automation supports approvers vs reviewers:** Focus on holistic
+    acceptance of contribution such as dependencies with other features,
+    backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations]
+- Assigned PRs to review related to project of expertise
+- Assigned test bugs related to project of expertise
+- Granted "read access" to submariner repo
```

Comment:
Why read access?, everybody has read access :-)

Reply:
Good question - I was thinking I should work with you prototype the mechanics on a test repo to verify things like this.

It's just copied from the K8s version, I'll remove it for now.